web/rootfs/defaults/ssl.conf

```diff
@@ -13,10 +13,11 @@ ssl_certificate_key /config/keys/cert.key;
 {{ end }}
 
 # protocols
+# Mozilla Guideline v5.4, nginx 1.17.7, OpenSSL 1.1.1d, intermediate configuration, no OCSP
+# https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=intermediate&openssl=1.1.0l&ocsp=false&guideline=5.4
 ssl_protocols TLSv1.2;
-ssl_prefer_server_ciphers on;
-ssl_ecdh_curve secp384r1;
-ssl_ciphers ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:ECDH+CHACHA20:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:!aNULL:!eNULL:!EXPORT:!DES:!RC4:!3DES:!MD5:!PSK;
+ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
```

Comment:
A note regarding `DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384`: Diffie-Hellman ciphers had been previously removed with https://github.com/jitsi/docker-jitsi-meet/pull/450 as discussed in https://github.com/jitsi/docker-jitsi-meet/issues/433
In case you add any DH ciphers again, it will also be necessary to add a `dhparams.pem` file again, which has also been removed with the PR mentioned above.

Reply:
Following Mozilla's advice, I'd like to deploy ffdhe2048 as the DH params, rather than generating them randomly.

However, I'm not so familiar with Docker configuration - what's the best place to install this, both in the repo and in the docker?

Reply:
Is that a file we'd like to include? In that case you can put it right on this directory, which will be available as /defaaults in the container.